packages/frontend/package.json

```diff
@@ -18,9 +18,14 @@
   },
   "scripts": {
     "start": "npm run clean && npm run build:watch",
-    "build": "cross-env NODE_ENV=development webpack",
-    "build:prod": "cross-env NODE_ENV=production webpack",
-    "build:watch": "cross-env NODE_ENV=development webpack --watch",
+    "build": "ng run frontend:build-elements:production",
+    "build:watch": "npm run build -- --watch",
```

Comment:
We should also have a `build:prod` to match the other project as well as ensure running `build:prod` from the root builds the frontend. Even if it just calls the `build`.

Reply:
Yeah that makes sense, I'll add that.